sponsorship.md

```diff
@@ -0,0 +1,46 @@
+# Pangeo Sponsorship Policy
+
+Now that Pangeo has a 501c3 fiscal sponsor, it can receive donations. Individuals or institutions who make donations to the project are **Project Sponsors**. The purpose of this document is to define the relationship between the Project and Project Sponsors.
```

Comment:
```suggestion
Now that Pangeo (here on out referred to as "The Project") has a 501(c)(3) fiscal sponsor, it can receive donations. Individuals or institutions who make donations to the project are **Project Sponsors**. The purpose of this document is to define the relationship between the Project and Project Sponsors.
```